server/src/main/java/org/elasticsearch/action/admin/indices/shards/IndicesShardStoresResponse.java

```diff
@@ -258,11 +258,9 @@ public void writeTo(StreamOutput out) throws IOException {
         }
 
         @Override
-        public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
-            builder.startObject();
-            builder.field("node", nodeId());
+        public XContentBuilder innerToXContent(XContentBuilder builder, Params params) throws IOException {
             super.innerToXContent(builder, params);
-            builder.endObject();
+            builder.field("node", nodeId());
```

Comment:
Can you move this before `super.innerToXContent(builder, params)` as it was before? The exception might be very verbose and this `node` field might be difficult to see if not placed first.